src/v/cloud_storage/partition_recovery_manager.cc

```diff
@@ -395,6 +400,11 @@ partition_downloader::download_log_with_capped_size(
           }
       });
     update_downloaded_offsets(std::move(dloffsets), dlpart);
+    if (dlpart.num_files == 0) {
```

Comment:
what does `num_files` mean in `download_part`?

Reply:
number of files in the `*_part` directory